core/src/main/java/com/google/instrumentation/trace/TraceExporter.java

```diff
@@ -75,6 +77,31 @@
   }
 
   /**
+   * Implementation of the {@link ServiceHandler} which logs all the exported {@link SpanData}.
+   *
+   * <p>Example of usage:
+   *
+   * <pre>{@code
+   * public static void main(String[] args) {
+   *   Tracing.getTraceExporter().registerServiceHandler(
+   *       "com.google.instrumentation.LoggingServiceHandler", new LoggingServiceHandler());
+   *   // ...
+   * }
+   * }</pre>
+   *
+   */
+  public static final class LoggingServiceHandler extends ServiceHandler {
```

Comment:
If this class is part of the API, I think it should have a factory method and a private constructor.